### DISCUSSION TITLE
Xiaomi WRS-R02 No Longer Working

### DISCUSSION THREAD

**Discussion by MercianWhitey** [Koenkk/zigbee2mqtt#11193] [Devices]
When we woke this morning our two Xiaomi WRS-R02 keypads had stopped working, they both worked fine before we went to bed. As far as I can tell, no part of our system has been updated. All my other Zigbee devices are working fine. The only thing I can find in the logs is the following:

Zigbee2MQTT:error 2022-02-02 17:21:09: Failed to configure 'Kitchen KP01', attempt 1 (Error: Bind 0x54ef4410002a1fed/1 genOnOff from '0x00124b00258d405a/1' failed (AREQ - ZDO - bindRsp after 10000ms)
    at Timeout._onTimeout (/app/node_modules/zigbee-herdsman/src/utils/waitress.ts:64:35)
    at listOnTimeout (internal/timers.js:557:17)
    at processTimers (internal/timers.js:500:7))

I've rebooted everything but that hasn't helped

Can anyone help?

Thanks

**Comment by SergiTello** [comment]
I have the same problem as you.
The WRS-R02s have stopped working, everything else works fine.

The only thing that I think could be related is the version of Zigbee2mqtt, with 1.23.0-1 I think they have stopped working.

Any idea or solution?

**Reply by MercianWhitey** [reply]
I hadn't realised that there had been an update, at least that explains why they stopped working! Still no solution though
